### ISSUE TITLE
Deselect state for province is not identified by Talkback

### ISSUE THREAD

**Issue by sarasuri** [cds-snc/covid-shield-mobile#571]
Samsung Galaxy J3 Prime
Android version 7.0

With Talkback on, on the "Step 6: Where do you live? (optional)" screen, users double tap to select a province and when it is selected Talkback announces that it is selected. However, when users double tap on the selected province to deselect, it doesn't announce it is deselected but the check is visually removed. Users wouldn't know if it is deselected.

Expected result
Talkback should identify the state of the radio button when deselected.

WCAG
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html

**Comment by qanitah** [comment]
Similar behaviour for iOS on iPhone 7. It announces 'selected' again with the province when user double taps to deselect the province with VoiceOver on.
